app/src/main/java/com/sdpteam/connectout/notifications/Constants.java

```diff
@@ -0,0 +1,9 @@
+package com.sdpteam.connectout.notifications;
+
+public class Constants {
```

Comment:
I don't see the point of having a class for these 3 constants. I would move them to the notification service class